inst/templates/tidy_contributing.md

```diff
@@ -0,0 +1,192 @@
+[the `#15`
+syntax](https://help.github.com/articles/autolinked-references-and-urls/)
+and the issue slug for context. If the PR is meant to close an issue,
+make sure one of the commit messages includes [text like `closes #44` or
+`fixes
+#101`](https://help.github.com/articles/closing-issues-using-keywords/).
+Provide the issue number and slug in the description, even if the issue
+is mentioned in the title, because auto-linking does not work in the PR
+title.
+
+  - GOOD PR title: *“Obtain user’s intent via mind-reading; fixes
+    \#86”.*
+  - BAD PR title: *“Fixes \#1043”.* Please remind us all what issue
+    \#1043 is about\!
+  - BAD PR title: *“Something about \#345”.* This will not actually
+    close issue \#345 upon merging.
+
+Add a bullet to `NEWS.md` with a concise description of the change, if
+it’s something a user would want to know when updating the package.
+[dplyr’s
+`NEWS.md`](https://github.com/tidyverse/dplyr/blob/master/NEWS.md) is a
+good source of examples. Note the sentence format, the inclusion of
+GitHub username, and links to relevant issue(s)/PR(s). We will handle
+any organization into sub-sections just prior to a release.
+
+What merits a bullet?
+
+  - Fixing a typo in the docs does not, but it is still awesome and
+    deeply appreciated.
+  - Fixing a bug or adding a new feature is bullet-worthy.
+
+When a maintainer does review a contribution, try to address the
+comments in short order, your changes are much more likely to be
+accepted if they are addressed in the next day than the next month.
+
+## On contributing
+
+### View contributing as a relationship, not a transaction
+
+The best way to be successful contributing to open source projects is to
+do so repeatedly. This means cultivating trust between yourself and the
+maintainer by multiple successful contributions. After a series of
+smaller contributions the maintainer will be much more willing to review
+and accept more substantial changes. As with any relationship being
+polite and considerate throughout will go a long way to improve trust.
+If you instead view the contribution as a solitary transaction to add
+your pet feature you are much less likely to be successful.
+
+### Code of Conduct
+
+Please note that this project is released with a [Contributor Code of
+Conduct](CONDUCT.md). By participating in this project you agree to
+abide by its terms.
+
+#### Further reading
+
+This document has been largely inspired by Jenny
+Bryan’s [googledrive](http://googledrive.tidyverse.org/)
+[`CONTRIBUTING.md`](https://github.com/tidyverse/googledrive/blob/master/CONTRIBUTING.md),
+Hadley Wickham’s [Contributing to ggplot2
+development](https://github.com/tidyverse/ggplot2/blob/92666ca8dd4cb5f96cbfcd4dcfcf157b599a6048/CONTRIBUTING.md),
+and Jim Hester’s [Contributing Code to the
+Tidyverse](http://www.jimhester.com/2017/08/08/contributing/), all of
+which are worth reading in their own right.
```

Comment:
I'm recommending a link to Jim's post already. We can probably drop the links to google drive and ggplot2, just to make this leaner.